AGENTS.md

```
@@ -44,6 +44,46 @@ tox -e ruff                  # lint
+```bash
+# Ask anything about structure, coverage, relationships
+graphify query "how does X work"
+graphify query "what calls Y"
+graphify path "AuthModule" "Database"
+graphify explain "OAuthClientCredentialsAuth"
+```
+
+Graphify is faster than grepping and surfaces cross-file relationships the raw code doesn't make obvious. Only fall back to `grep`/`Read` when the graph answer is insufficient.
+
+To rebuild the graph after large changes:
+```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)